psalmtone.js

```diff
@@ -260,8 +267,13 @@ var o_g_tones =
                    },
              'V.2':{clef:"c4",
                     mediant:"hr h h/hf,fgwhvGFEfg/gf"
-                   }
-            };
+                   },
+			
+  'V. Solemnior':{clef:"c3",
+         mediant:"hr 'hi hr h_, hr fe 'f_h hr hiHGhih.ghGFE'fggf."

```

Comment:
Wouldn't it be more useful to put it in with a mediant and a termination:
```suggestion
         mediant:"hr 'hi hr h_,"
         termination: "hr f e 'f_h hr hiHGhih.ghGFE'fggf."
```

and then mark where the quarter bar goes in the text with an asterisk?